Content.Client/LateJoin/LateJoinGui.cs

```diff
@@ -24,9 +22,12 @@ public sealed class LateJoinGui : DefaultWindow
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
         [Dependency] private readonly IClientConsoleHost _consoleHost = default!;
         [Dependency] private readonly IConfigurationManager _configManager = default!;
+        [Dependency] private readonly IEntitySystemManager _entitySystem = default!;
 
         public event Action<(EntityUid, string)> SelectedId;
 
+        private readonly SpriteSystem _sprites = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<SpriteSystem>();
```

Comment:
cache this properly in the ctor like gameticker is

Reply:
What do you mean with properly caching? And to me gameticker doesn't look like it is cached at all.